api/stream.go

```diff
@@ -130,6 +133,47 @@ func pauseStream(c *gin.Context) {
 	}
 }
 
+// reportStreamIssue sends a notification to a matrix room that can be used for debugging technical issues.
+func reportStreamIssue(c *gin.Context) {
+	foundContext, exists := c.Get("TUMLiveContext")
+	if !exists {
+		sentry.CaptureException(errors.New("context should exist but doesn't"))
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+	tumLiveContext := foundContext.(tools.TUMLiveContext)
+	stream := tumLiveContext.Stream
+
+	lectureHall, err := dao.GetLectureHallByID(stream.LectureHallID)
+	if err != nil {
+		sentry.CaptureException(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+	}
+	course, err := dao.GetCourseById(c, stream.CourseID)
+	if err != nil {
+		log.WithError(err).Error("Can't send bot message")
+		sentry.CaptureException(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+	}
+	streamUrl := tools.Cfg.WebUrl + "/w/" + course.Slug + "/" + strconv.Itoa(int(stream.ID))
```

Comment:
fmt.Sprintf could be used to avoid string conversion here

Reply:
Didn't do any benchmarks on this yet, nice to know, thanks! This is not a hot path thought, so imo optimising for readability is fine

Reply:
Yes, readability is way more important here.